linera-core/src/lib.rs

```diff
@@ -22,4 +22,7 @@ pub mod worker;
 pub(crate) mod updater;
 pub(crate) mod value_cache;
 
+// TODO: Move to env vars or config.
```

Comment:
Let's create an issue for it and link to it here.
(Or compute the batch size from the gRPC message limit and the block size limit?)

Reply:
I'm planning to address this as part of the PR - i.e. so that client<>server adapt to the gRPC message limit size.

Reply:
Done.